README.md

```diff
@@ -1 +1,73 @@
-# Ben_Portfolio
\ No newline at end of file
+# Ben's Portfolio
+
+> This is my portfolio as a Microverse software engineer student
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://ben-microverse-project.com)
+
+## Getting Started
+
+To clone this project:
+- Go to github project repository
+- Click on the green button(known as code)
+- Copy the ssh link
+- on your local machine, type the command git clone link
+
+
+### Prerequisites
+- Code editor
+- Knowledge in HTML and CSS
+
+### Setup
+- No set up is required for this project
+
+### Install
+- No Installation is required
+
+### Usage
+- Make changes to the index.html and submit pull requests
+
+### Run tests
+- You can use a live server to run the tests
+
+### Deployment
+- Run the project in your local browser to view the index.html
+
+
+
+## Authors
+
+👤 **Ben Omayio**
+
+- GitHub: [@kiboma2021](https://github.com/kiboma2021)
+- Twitter: [@BenOmayio](https://twitter.com/omayiobenj)
+- LinkedIn: [@BenOmayio](https://www.linkedin.com/in/ben-omayio-74622469/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/BenOmayio/HelloMicroverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._

- [ ] Please fix the broken link in the license.